record/record.app.js

```diff
@@ -121,10 +122,16 @@
                     };
                     // return model;
                     callback(model);
+                    errorData.redirectUrl = $rootScope.reference.unfilteredReference.contextualize.compact.appLink;
+                    errorData.gotoTableDisplayname = $rootScope.reference.displayname.value;
+                    error.errorData = errorData;
```

Comment:
Lines 125-127 are unnecessary here. When this error is thrown on line 128, it is caught by the `catch` statement right below it where you set all of these values again.